src/main/java/io/grayray75/fabric/fpsdisplay/mixin/InGameHudMixin.java

```diff
@@ -22,8 +22,9 @@ public void render(MatrixStack matrixStack, float tickDelta, CallbackInfo info)
         if (!client.options.debugEnabled && config.enabled && config.textAlpha > 3 && FpsDisplayMod.SHOW_FPS_OVERLAY) {
 
             String displayString = ((MinecraftClientMixin) client).getCurrentFPS() + " FPS";
-            float textPosX = 2;
-            float textPosY = 2;
+            int guiScale = client.options.guiScale;
+            float textPosX = config.offsetLeft / guiScale;
+            float textPosY = config.offsetTop / guiScale;
```

Comment:
This will cause a crash if the GUI scale is set to 'auto', because then `client.options.guiScale` has the value 0

Reply:
Oh, I didn't test for that, thanks! In this case maybe it should use the values as is without scaling, what do you think?

I would change the code to this and add it as a commit to this PR:
```java
float textPosX = config.offsetLeft;
float textPosY = config.offsetTop;

int guiScale = client.options.guiScale;
if (guiScale > 0) {
  textPosX /= guiScale;
  textPostY /= guiScale;
}
```